standard/expressions.md

```diff
@@ -3575,6 +3653,24 @@ The result of evaluating `~x`, where `X` is an expression of an enumeration ty
 
 Lifted ([§12.4.8](expressions.md#1248-lifted-operators)) forms of the unlifted predefined bitwise complement operators defined above are also predefined.
 
+### §hat-operator Hat operator
+
+The unary `^` operator is called the *hat* operator. The hat operator is not overloadable (§12.4.3) and there is a single predefined hat operator:
```

Comment:
I'm still not thrilled with "hat operator". While I'm not sold on any of these, here are some alternatives:

- "end operator"
- "reverse index operator"
- "from end operator" (I like this one least)
- "tail end operator"

Reply:
Interestingly, I like "from end operator" *most* as it corresponds with `Index.FromEnd`. (I could live with "index-from-end operator" as well, to be even clearer...)

Reply:
I'm most familiar personally with "the from-end operator" from design discussions, and I prefer it the most due to it describing exactly what it does.

Reply:
Defer to a new issue. We considered "The unary `^` operator is called the *from-end* operator. (It is also sometimes known as the *hat* operator.)" That would aid searchability.

Reply:
Spun off as #1404 and this conversation marked as resolved.